### PR TITLE
[DEV-5104] Add Sentry error reporting

### PR DESCRIPTION
**Pull request by oleg-semyonov**

### CODE REVIEW

File: modules/Stories/StoryCard.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const StoryCard: FunctionComponent<Props> = ({ story }) => {
                         <div className="mt-6 flex items-center">
                             <div className="flex space-x-1 text-sm text-gray-500">
                                 <time dateTime="2020-03-16">
```
Comment (kudlajz) [inline modules/Stories/StoryCard.tsx:78]: Why is the date hardcoded here? 😄  I'd say that if `publicationDate` is undefined then it doesn't make sense to render this element at all. And if it is defined, then probably render the correct date there.

File: utils/discourse/syncDiscourseThread.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { FormatVersion } from '@prezly/sdk/dist/types/Story';
+import { StoryFormatVersion } from '@prezly/sdk/dist/types/Story';
```
Comment (kudlajz) [inline utils/discourse/syncDiscourseThread.tsx:1]:
```suggestion
import { StoryFormatVersion } from '@prezly/sdk';
```

File: modules/Story/Story.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import type { FunctionComponent } from 'react';
 import type { ExtendedStory } from '@prezly/sdk/dist/types';
 import SlateRenderer from 'components/SlateRenderer';
-import { FormatVersion } from '@prezly/sdk/dist/types/Story';
+import { StoryFormatVersion } from '@prezly/sdk/dist/types/Story';
```
Comment (kudlajz) [inline modules/Story/Story.tsx:4]:
```suggestion
import { StoryFormatVersion } from '@prezly/sdk';
```

**Review by kudlajz — APPROVED** [review]

File: .github/workflows/build-and-deploy.yml [hunk 4]
```diff
@@ -34,12 +34,23 @@ jobs:
...
+            "NEXT_PUBLIC_SENTRY_DSN=${{ secrets.NEXT_PUBLIC_SENTRY_DSN }}"
+    - name: Clear cache
+      id: clear-cache
       run: |
-        docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:${{ steps.prep.outputs.BUILD_ID }} .
-        docker push $ECR_REGISTRY/$ECR_REPOSITORY:${{ steps.prep.outputs.BUILD_ID }}
\ No newline at end of file
+          sleep 90
+          curl --request POST \
+              --url https://admin-api.prezly.com/v1/themes/7ae88112-150b-4475-b38f-bc10872a396f/clear-cache \
```
[the remark this answers is not in the document]
Comment (oleg-semyonov) [reply .github/workflows/build-and-deploy.yml:54]: Yeah, it's seems to be Bea ID. Thanks for the catch, I'll update it!

**Pull request merged**